rest-api/edit-conversation-image-title/server/server.js

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
+import cors from 'cors';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const app = express();
+app.use(cors())
+app.use(bodyParser.json())
+
+async function editImageOrTitle(conversationId, imageURL, conversationTitle) {
+  
+  const data = conversationTitle === "" ? { photoUrl: imageURL } : imageURL === "" ? { subject: conversationTitle } : { subject: conversationTitle, photoUrl: imageURL };
+
+  const talkJSURL = `${process.env.TALKJS_URL}/${process.env.APP_ID}/conversations/${conversationId}`;
+  const options = {
+    method: "PUT",
+    headers: {
+      Authorization: `Bearer ${process.env.SECRET_KEY}`,
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(data),
+  };
+  try {
+    const response = await fetch(talkJSURL, options);
+    console.log(response);
```

Comment:
It'd be better to specifically log the thing you're interested in - ie the body or status code etc. I don't think logging the response object will be very useful